nano/core_test/utility.cpp

```diff
@@ -159,7 +178,7 @@ TEST (thread_pool_alarm, one)
 	nano::condition_variable condition;
 	workers.add_timed_task (std::chrono::steady_clock::now (), [&] () {
 		{
-			nano::lock_guard<nano::mutex> lock{ mutex };
+			nano::lock_guard<nano::mutex> lock (mutex);
```

Comment:
There is a bunch of lock mutex contractions here that look like they were made by mistake.